content/webapp/services/prismic/transformers/embeds.ts

```diff
@@ -19,6 +19,8 @@ export function transformVideoEmbed(
     return {
       embedUrl: getYouTubeEmbedUrl(embed),
       videoProvider: 'YouTube',
+      videoThumbnail: embed.thumbnail_url || undefined,
+      title: embed.title || '',
```

Comment:
Could we be using `https://github.com/wellcomecollection/wellcomecollection.org/blob/main/content/webapp/services/prismic/transformers/body.ts#L327` or does it have to be different? I guess it doesn't have title, but could we have added it in?

Reply:
That one also has no thumbnail though 🤔 be nice to merge them if we can I guess, so we always pull the same info.